cvat/apps/engine/models.py

```diff
@@ -677,6 +679,7 @@ class Job(TimestampedModel):
 
     type = models.CharField(max_length=32, choices=JobType.choices(),
```

Comment:
Please add a new job type in the enum.

Reply:
But then we would also have to check that that there isn't any logic based on job type due to which the consensus job doesn't work properly.

Like some functionalities which it exhibited since it was inheriting the job type, would now not work, and break?

Reply:
Despite that, shouldn't the job type structure look like this?

- Job Types
  - Ground Truth
  - Annotation
     - Normal Job
     - Consensus Job

And inevitably the information about whether it's a normal or consensus job is present in parameter `parent_job_id`...?

Reply:
> Despite that, shouldn't the job type structure look like this? [...]

I don't see a point in making such a nested structure at this point. The consensus jobs are not the same as annotation jobs, they have different handling logic in annotation import and export. Probably, they should not be included in the task quality report. It's better to just keep the type list flat, unless there is a clear use case for such a structure.

> But then we would also have to check that that there isn't any logic based on job type due to which the consensus job doesn't work properly.
> Like some functionalities which it exhibited since it was inheriting the job type, would now not work, and break?

You would need to check this in any case, as these jobs aren't the same as "normal" jobs.